meilisearch-http/tests/common/server.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
Why is there a `#[allow(dead_code)]` here? Can't the `get_dump_status` method be conditionally compiled?

Reply:
this is because the module is defined in the test file, and if the test file doesn't use all of the defined methods in the module, I get warnings. Overall all the methods are used, but in different test files.